ramls/course.json

```diff
@@ -45,6 +45,10 @@
             "type": "string",
             "description": "A text field denoting this course's section"
         },
+        "numberOfStudents": {
+            "type": "integer",
+            "description": "The number of students enrolled in this listing"
```

Comment:
This is a course field, but the comment says it belongs to the listing.

Reply:
Good catch, the location changed during development and I forgot to update it. Fixed now!